### DISCUSSION TITLE
not finding scurity vulnerabilities

### DISCUSSION THREAD

**Discussion by vikisec** [projectdiscovery/nuclei#3168] [Q&A]
nuclei is not finding security vulnerabilities in a vulnerrable website. tried it with different websites but same result. refer the screen shot attached below
![Screenshot from 2023-01-06 11-00-55](https://user-images.githubusercontent.com/91051443/210937013-483312f1-8e7b-4c34-83a2-733e9f0b3b7c.png)

**Comment by geeknik** [comment]
It looks like you're using Nuclei 2.8.3, however, Nuclei 2.8.6 is the latest release version, I'd upgrade to that and try again.

**Comment by vikisec** [comment]
![1](https://user-images.githubusercontent.com/91051443/211022556-ec7a9094-039e-4ea4-894e-81c763ceb9a6.png)
same output with latest release still not finding bugs

**Reply by vancong160202** [reply]
Have you fixed to detect the vulnerability with juice shop web? If so, please help me how to do it. Thank you

**Comment by geeknik** [comment]
I wish I could say I could reproduce the behavior, but my template testing ground shows nuclei appears to be working as intended...

![image](https://user-images.githubusercontent.com/466878/211027803-fb299f33-39c0-4bf5-b5d6-8ebff4626c5b.png)

![image](https://user-images.githubusercontent.com/466878/211028344-0864e366-5a6a-4d8f-a66b-8454d8edaf30.png)

**Comment by vikisec** [comment]
the diffrence is thatt you are loading fuzzing templates in first. i am just giving a target and let all templates scan. just do a test with all templates with "./nuclei - target www.domain.com". and upload the result.

**Comment by geeknik** [comment]
Second screenshot is 2583 templates from the 9.3.4 release minus the boring info/low severity ones. I have nothing more to add. Someone else should chime in.

**Comment by ehsandeep** [comment]
@vikisec nuclei find vulnerability based on the template you were running, and I don't see the target website has any vulnerability that can be detected from the default set of nuclei templates; some of them can be found using fuzzing templates, and you can write more custom template to find more.

**Reply by vikisec** [reply]
actually i  tested on "http://testphp.vulnweb.com/" for target practise, this website had a lot of vulnerrabilities. i scanned it before(few months ago) and find vulnerabilities. now i also test on "https://juice-shop.herokuapp.com/#/" it also had same vulnerabilities but not finding anything on nuclei. but when i scan now all these websites are showing clean output in nuclei . i am sure something wrong with nuclei & thats why i reported.

**Reply by olearycrew** [reply]
@vikisec when I go to https://juice-shop.herokuapp.com/#/ there is no application there - just the generic Heroku page. It's possible that it was part of Heroku's free tier and thus is [gone now](https://techcrunch.com/2022/08/25/heroku-announces-plans-to-eliminate-free-plans-blaming-fraud-and-abuse/).

**Reply by duhang1996** [reply]
I encountered the same situation as  @vikisec

**Reply by duhang1996** [reply]
Have you solved it yet @vikisec
